source/Features.rst

```diff
@@ -564,7 +579,7 @@ Click on the “Submit” button. Now You can connect to the SSH Terminal in a n
 S3 Product
 -----------
 
-Researcher can login to the portal and quickly order S3 Product.
+As a Researcher,You can login to the Research Gateway and quickly order S3 Product.
```

Comment:
Needs space after comma and the following word after that should begin with lowercase if it is not a proper noun.